tests/scripts/thread-cert/node.py

```diff
@@ -2724,6 +2724,10 @@ def ping(self, *args, **kwargs):
         else:
             return super().ping(*args, **kwargs)
 
+    def udp_send_host(self, ipaddr, port, data, multicast_hop_limit=10):
```

Comment:
```suggestion
    def udp_send_host(self, ipaddr, port, data, hoplimit=None):
```

This command should work for unicast udp send too?
Maybe we can set default hoplimit to 10 for multicast and 64 for unicast?